### PR TITLE
RWA-879 fix functional tests in CFT action and search controllers

### PR DESCRIPTION
**Pull request by vgulapally**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-879

### Change description ###
fix functional tests in CFT action and search controllers

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by alexbaeza — APPROVED** [review]
LGTM

**Review by dharmendrak — DISMISSED** [review]
LGTM

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java [hunk 1]
```diff
@@ -448,6 +448,35 @@ public void should_return_a_200_with_search_results_based_on_state_unassigned()
...
+            searchTaskRequest,
+            authenticationHeaders
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("tasks.size()", lessThanOrEqualTo(10))
+            .body("tasks.jurisdiction", everyItem(is("IA")))
+            .body("tasks.task_state", everyItem(is("assigned")))
+            .body("total_records", greaterThanOrEqualTo(1));
```
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java:475]: We might be able to also check the taskRoleResource has some expected values?

Comment (vgulapally) [reply src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java:475]: not needed as discussed

**Review by alexbaeza — APPROVED** [review]

**Pull request merged**